packages/broker/test/integration/plugins/operator/deploySponsorshipContract.ts

```diff
@@ -1,39 +1,45 @@
 // TODO: copy-paste from network-contracts, import from there?
 import { utils, Wallet, Contract, ContractReceipt } from 'ethers'
 
+import { Logger } from '@streamr/utils'
+
 import { sponsorshipABI, sponsorshipFactoryABI } from '@streamr/network-contracts'
 import type { Sponsorship, SponsorshipFactory } from '@streamr/network-contracts'
 
 const { parseEther } = utils
 
+const logger = new Logger(module)
+
 export async function deploySponsorship(
     // eslint-disable-next-line max-len
-    chainConfig: { contracts: { SponsorshipFactory: string, SponsorshipStakeWeightedAllocationPolicy: string, SponsorshipDefaultLeavePolicy: string, SponsorshipVoteKickPolicy: string } },
+    chainConfig: { contracts: { SponsorshipFactory: string, SponsorshipStakeWeightedAllocationPolicy: string, SponsorshipDefaultLeavePolicy: string, SponsorshipVoteKickPolicy: string, StreamRegistry: string } },
     deployer: Wallet, {
         streamId = `Stream-${Date.now()}`,
         metadata = '{}',
         minimumStakeWei = parseEther('60'),
         minHorizonSeconds = 0,
         minOperatorCount = 1,
+        earningsPerSecond = parseEther('0.01'),
     } = {},
 ): Promise<Sponsorship> {
+    const { contracts } = chainConfig
+    logger.debug('deploySponsorship', { contracts, deployerBalance: await deployer.getBalance() })
+
```

Comment:
The deployer should always have enough balance in the test environment? (Or if not in some test, the test will fail)

```suggestion
```

Reply:
Removed the log. The contracts const is needed further down